integration-cli/docker_cli_run_test.go

```diff
@@ -2559,25 +2559,72 @@ func (s *DockerSuite) TestRunWriteFilteredProc(c *check.C) {
 	}
 }
 
+func createTmpFile(c *check.C, content string) string {
```

Comment:
I looked through the other test cases. I could not find one that would have a code sequence taht creates a temp. file and writes contents into it.